src/lib/Scenes/Home/index.tsx

```diff
@@ -26,12 +26,43 @@ interface Props {
   selectedTab?: number
 }
 
-export default class Home extends React.Component<Props, null> {
+interface State {
+  appState: string
+}
+
+export default class Home extends React.Component<Props, State> {
+  tabView?: ScrollableTabView | any
```

Comment:
This typing `| any` also seems weird to me...but I don't know enough to really say more. 👻